notebooks/238-typo-detector/238-typo-detector.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>it is preferred to use python API for Model Optimizer then CLI, could you please convert model using this way?</p><p>see https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html for details</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Done. Please check